components/Builder.tsx

```diff
@@ -157,6 +159,10 @@ export default function Builder({ data, formation: _formation }: Props) {
     }
   }
 
+  function onResetFormation() {
+    setFormation(new Array<string>(13).fill(""));
```

Comment:
we want this to actually fill the formation with the appropriate length array based on the currently selected formation